src/showstyle0/part-adapters/dve.ts

```diff
@@ -72,43 +140,69 @@ export function generateDVEPart(context: PartContext, part: PartProps<DVEProps>)
 		adlibPreroll: SUPER_SOURCE_LATENCY,
 		content: {
 			...dveLayoutToContent(config, { boxes }, part.payload.inputs),
+			timelineObjects: dvePieceTimelineObjects,
+		},
+	}
 
-			timelineObjects: [
-				...createVisionMixerObjects(config, 6000, SUPER_SOURCE_LATENCY), // TODO - use config for vmix
-
-				literal<TSR.TimelineObjAtemSsrcProps>({
-					id: '',
-					enable: { while: 1 },
-					priority: 1,
-					layer: AtemLayers.AtemSuperSourceProps,
-					content: {
-						deviceType: TSR.DeviceType.ATEM,
-						type: TSR.TimelineContentTypeAtem.SSRCPROPS,
-						ssrcProps: parseSuperSourceProps(context, part.payload),
+	const retainPieceTimelineObjects: TimelineObjectCoreExt[] = []
+	if (config.visionMixerType === VisionMixerType.Atem) {
+		retainPieceTimelineObjects.push(
+			literal<TSR.TimelineObjAtemSsrcProps>({
+				id: '',
+				enable: { while: 1 },
+				priority: 0.5,
+				layer: AtemLayers.AtemSuperSourceProps,
+				content: {
+					deviceType: TSR.DeviceType.ATEM,
+					type: TSR.TimelineContentTypeAtem.SSRCPROPS,
+					ssrcProps: {
```

Comment:
I think these should be the same props as the normal piece.

Reply:
Fixed